client/header/style.scss

```diff
@@ -33,6 +35,15 @@
 		line-height: $header-height;
 		background: $studio-white;
 
+		> p.is-embed {
```

Comment:
This one shows up a bit funky to me in the CSS.  Also curious why one would be `13px` and one would be `14px`.  Do these show up differently on embed vs non-embed pages?

Reply:
The `font-size:13;` line is actually just a obsolete style at this point, since the `<Text>` component now used for that header assigns its own styles (which match the embed style at 14px). I've been leaning towards not removing styles to ensure I don't remove anything that's being used in some unexpected way, but I think it's safe to do away with these. 

I'm not sure what you mean regarding the funkiness; I'll ping you on the slacks.

Reply:
On second thought, to simplify this a bit, I think we can remove the `is-embed` class altogether and keep the font styles on the `.woocommerce-layout__header-heading` selector.  Then we can remove the wrapping `p` tag below.

```
			font-weight: 600;
			font-size: 14px;
```